projects/archiver/src/issueTask.ts

```diff
@@ -0,0 +1,41 @@
+import { Handler } from 'aws-lambda'
+import { head, tail } from 'ramda'
+import { attempt, hasFailed } from '../../backend/utils/try'
+import { run } from '../main'
+export interface Issue {
+    source: string
+    id: string
+}
+export interface IssueParams {
+    issues: Issue[]
+    parsedIssues?: Issue[]
+}
+export const handler: Handler<
+    IssueParams,
+    IssueParams & {
+        parsed?: Issue
+        success: boolean
+        message?: string
+        finished: boolean
+    }
+> = async ({ issues, parsedIssues }) => {
+    //Should not be called with empty issues, but short circuit if so.
+    if (issues.length == 0)
+        return { issues, parsedIssues, success: true, finished: true }
+
+    const current = head(issues)
+    const remaining = tail(issues)
+
+    const publish = await attempt(run(current.source, current.id))
```

Comment:
In this context I think we should change the function name from `run` to something a bit more descriptive. Even if it's just `runArchiver`. There are a lot of small, named function here and it sort of gets lost on first glance that this is where the business logic is happening. Not a big issue but it might help it to leap out.

Reply:
Pulling the logic out of `run` asap.